.github/ISSUE_TEMPLATE.md

```diff
@@ -1,58 +1,20 @@
-#### Do NOT include any confidential information, this issue will be created on a public repo
-#### This section is completed by the issue reporter
-##### On which version(s) do you encounter this issue?
-###### Please provide the output to 'git describe --all' for  all deployments where you have confirmed it occurs.
-
-##### What type of issue is this? Please tick the appropriate box.
-- [ ] bug - existing functionality that does not work for you as documented
-- [ ] feature - new functionality that you would like to see added
-
-##### What is the expected behavior?
-
-##### What is actually happening?
-
-##### What were you attempting to do when you encountered this error (give as much detail as possible)? 
-
-##### How can we reproduce this issue?
-
-##### How can we verify your issue is addressed?
-
-##### If you are using or know of a workaround please describe the necessary steps?
-
-##### What is your evaluation on the importance of this issue, and why?
-
-----------------------------------------------------------------------------------------------------------------------------
-#### This section is completed during the issue resolution process. If you are reporting an issue, please do not modify the section below.
-##### Which branches are impacted by this issue?
-###### Remove branches that are not impacted.
-###### For each pull request, update the branch line to reference it.
+#### Which version of rpc-openstack does this issue relate to?
```

Comment:
I'd remove these check boxes and simply replace them with a required text field denoting the actual SHA they're on. This can be easily achieved with the following:

``` bash
git log --pretty=oneline | head -n 1
```

Or maybe just:
``` bash
git status | head -n 1
```

This output would give us the exact point in time the bug reporter is starting from. Regardless of branch or tag we'd be able recreate the issues from the exact point in time.